lib/vapi-client-bindings/index.html

```diff
@@ -1 +1 @@
-<a href='vapi_client_bindings-3.2.0-py2.py3-none-any.whl'>vapi_client_bindings-3.2.0-py2.py3-none-any.whl</a><br />
```

Comment:
remove trailing space

Reply:
This file is edited by the generator. However, I couldn't find the trailing space in the editor.

Reply:
yes, ignore it